### PR TITLE
feat: add WhyWeDoIt component

### PR DESCRIPTION
**Pull request by AbdallahSafar**
Screenshot (415)

For testing use: `<WhyWeDoIt />`

### CODE REVIEW

**Comment by AbdallahSafar** [comment]
Screenshot (417)_LI

App name and icon

File: src/components/WhyWeDoIt/WhyWeDoIt.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import heart from "./like-heart.svg";
+
+function WhyWeDoIt() {
+  return (
+    <div className="flex flex-col items-center w-52 h-64 shadow-md p-4 rounded-lg border">
```
Comment (osamaakb) [inline src/components/WhyWeDoIt/WhyWeDoIt.jsx]: width should be 100%

**Review by osamaakb — CHANGES_REQUESTED** [review]
Good job, take a look at the comments

File: src/components/WhyWeDoIt/WhyWeDoIt.jsx (hunk 1 above)
Comment (AbdallahSafar) [reply src/components/WhyWeDoIt/WhyWeDoIt.jsx]: What do you mean 100%? Shouldn't be like figma? @osamaakb

Comment (osamaakb) [reply src/components/WhyWeDoIt/WhyWeDoIt.jsx]: it should be one under one, I left comments for changes to be done to Figma, but no one did the changes so let us do it here

Comment (AbdallahSafar) [reply src/components/WhyWeDoIt/WhyWeDoIt.jsx]: Screenshot (422)
You mean like that right but a bit larger or a card with a full width ?

**Comment by AbdallahSafar** [comment]
Screenshot (423)

@osamaakb Better?

**Comment by osamaakb** [comment]
Looks good

**Comment by osamaakb** [comment]
however, this is not 100%, this is `w-72` but I will fix it

**Review by osamaakb — APPROVED** [review]

**Pull request merged**